.github/workflows/deploy_storybook.yml

```
@@ -0,0 +1,38 @@
+name: Test Build and Deploy Storybook to Vercel
+
+on:
+  push:
+    branches:
+      - master
+      - storybook_deploy
+    paths:
+      - packages/components/**
+
+jobs:
+  deploy_storybook:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
```

You can use `actions/setup-node@v2` here.

```suggestion
        uses: actions/setup-node@v2
```